Doc/library/sys.rst

```diff
@@ -97,7 +117,7 @@ always available.
    native function is preferred when possible.
 
    See the :ref:`audit events table <audit-events>` for all events raised by
-   CPython.
+   CPython, and :pep:`578` for the original design discussion.
```

Comment:
2nd mention of the PEP.  I think 1 would be enough.

Reply:
Fixed!